ITEventsTests/Info.plist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I am not sure if this file is really needed? Can you please check?

Reply:
It is default line. I can make changes only in dict